lib/pages/settings.dart

```diff
@@ -365,6 +363,28 @@ class _SettingsPageState extends State<SettingsPage> {
                                     })
                               ],
                             ),
+                            installMethodDropdown,
+                            height8,
+                            Text(
+                                tr('installMethodsExplanation'),
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .labelSmall
+                            ),
+                            height8,
+                            GestureDetector(
```

Comment:
I think the explanation and about link are unnecessary - they are only relevant to users who would likely not need them.